source/_integrations/kira.markdown

```diff
@@ -83,7 +87,11 @@ remotes:
 
 If no sensors or remotes are specified, a sensor with default values will be added.
 
-### Code Configuration
+### Configuration
+
+Restart your Home Assistant server and you should now have an entity called kira_remote (or whatever you called it). To check go to Developer Tools > States and start to type “kira” within the entity list. 
+
+![alt text](https://www.info.keene-electronics.co.uk/kira-home-assistant/img/Screenshot%202020-03-18%2016.26.12.png "kira_dev_tools")
```

Comment:
We should host the screenshot.

Reply:
Agreed I would prefer it too. I have created an image repository on github and linked from there. If there is a better way please advise.

Reply:
The image needs to be added to this repository.

Reply:
I have amended the image links, is this OK?